tests/test_modeling_flax_common.py

```diff
@@ -1026,6 +1026,12 @@ def test_no_automatic_init(self):
             inputs = self._prepare_for_class(inputs_dict, model_class).copy()
             model(**inputs, params=params)
 
+            # test apply
```

Comment:
Fine to make this a new standalone test! It's quite fast to run the tester mixin

Reply:
Wonder if there's a way we could pre-determine / indicate which modules are apply ready?

Reply:
Sound good I'll try to make a new test.
I am very curious if `model.module.__call__` can maintain the same signature as `model.__call__` in all cases? It seems to work for BERT but it might be a coincidence.

Reply:
I reckon so - we tend to do a bit of pre-processing in `model.__call__`, and a bit in `model.module.__call__`. Moving all of this to the latter should be possible for the bulk of the Flax models in the library (albeit some work!)